packages/vite-plugin-svelte/src/utils/preprocess.ts

```diff
@@ -1,14 +1,7 @@
-import {
-	transformWithEsbuild,
-	ESBuildOptions,
-	ResolvedConfig,
-	TransformResult,
-	Plugin
-} from 'vite';
+import { transformWithEsbuild, ESBuildOptions, ResolvedConfig, Plugin } from 'vite';
```

Comment:
can we use  `import * as vite from 'vite'` here and then skip the dynamic import below?
And maybe be specific about type imports.

Reply:
Good idea, didn't thought of that

Reply:
updated in f83c46c